src/Dinosaur/Dinosaur.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace DinoApp\Dinosaur;
+
+class Dinosaur
+{
+    protected int $id;
+    protected string $species;
+    protected string $foodType;
+    protected float $height;
+    protected float $weight;
+    protected float $length;
+    protected int $killerRating;
+    protected int $intelligence;
+    protected int $age;
+    protected string $imageUrl;
+
+//    /** Class Constructor made for the purposes of unit testing
```

Comment:
Eventually need to pull in s2t2 with the default values for the constructor so this can be uncommented but seems like you're on this